src/CoderDojo/WebsiteBundle/Form/Type/CocRequestFormType.php

```diff
@@ -50,13 +50,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => 'Achternaam'
             ))
             ->add('email', EmailType::class, [
-                'label' => 'Email'
+                'label' => 'Email',
+                'attr' => [
+                    'placeholder' => 'Let op, dit moet een persoonlijk emailadres zijn.'
```

Comment:
yes, or `sander@coderdojo-city.nl`... Since the government needs to contact the volunteers personally and it is going through DigID we need to have personal email addresses. This also helps us to determine if someone already had a VOG through a different dojo (in case a volunteer is active in multiple dojo's)